Zend/zend_vm_def.h

```diff
@@ -7234,6 +7346,7 @@ ZEND_VM_HANDLER(149, ZEND_HANDLE_EXCEPTION, ANY, ANY)
 	if (throw_op->result_type & (IS_VAR | IS_TMP_VAR)) {
 		switch (throw_op->opcode) {
 			case ZEND_ADD_ARRAY_ELEMENT:
+			case ZEND_ADD_ARRAY_UNPACK:
```

Comment:
zend_opcode.c also needs updates (add ADD_ARRAY_UNPACK next to ADD_ARRAY_ELEMENT).